source/usr/lib/mcvirt/node/node.py

```diff
@@ -16,51 +18,45 @@
 # along with MCVirt.  If not, see <http://www.gnu.org/licenses/>
 
 import re
+import Pyro4
 
-from mcvirt.mcvirt import MCVirtException
+from mcvirt.exceptions import (InvalidVolumeGroupNameException,
+                               InvalidIPAddressException)
 from mcvirt.mcvirt_config import MCVirtConfig
+from mcvirt.rpc.pyro_object import PyroObject
 
 
-class InvalidVolumeGroupNameException(MCVirtException):
-    """The specified name of the volume group is invalid"""
-    pass
-
-
-class InvalidIPAddressException(MCVirtException):
-    """The specified IP address is invalid"""
-    pass
-
-
-class Node(object):
+class Node(PyroObject):
+    """Provides methods to configure the local node."""
 
-    @staticmethod
-    def setStorageVolumeGroup(mcvirt_instance, volume_group):
-        """Update the MCVirt configuration to set the volume group for VM storage"""
+    @Pyro4.expose()
+    def set_storage_volume_group(self, volume_group):
+        """Update the MCVirt configuration to set the volume group for VM storage."""
         # Ensure volume_group name is valid
         pattern = re.compile("^[A-Z0-9a-z_-]+$")
         if (not pattern.match(volume_group)):
             raise InvalidVolumeGroupNameException('%s is not a valid volume group name' %
                                                   volume_group)
 
         # Update global MCVirt configuration
-        def updateConfig(config):
+        def update_config(config):
             config['vm_storage_vg'] = volume_group
-        mcvirt_config = MCVirtConfig(mcvirt_instance=mcvirt_instance)
-        mcvirt_config.updateConfig(updateConfig, 'Set virtual machine storage volume group to %s' %
-                                                 volume_group)
+        mcvirt_config = MCVirtConfig()
+        mcvirt_config.update_config(update_config,
+                                    'Set virtual machine storage volume group to %s' %
+                                    volume_group)
 
-    @staticmethod
-    def setClusterIpAddress(mcvirt_instance, ip_address):
-        """Updates the cluster IP address for the node"""
+    def set_cluster_ip_address(self, ip_address):
+        """Update the cluster IP address for the node."""
         # Check validity of IP address (mainly to ensure that )
         pattern = re.compile(r"^((([01]?[0-9]?[0-9]|2[0-4][0-9]|25[0-5])[ (\[]?(\.|dot)"
                              "[ )\]]?){3}([01]?[0-9]?[0-9]|2[0-4][0-9]|25[0-5]))$")
         if not pattern.match(ip_address):
             raise InvalidIPAddressException('%s is not a valid IP address' % ip_address)
```

Comment:
whoops (to the comment). This would be a good improvement... could you create a ticket?